src/applications/mhv-landing-page/components/ErrorNavCard.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+
+export const externalLinkText = '(opens in new tab)';
+import { CONTACTS } from '@department-of-veterans-affairs/component-library/contacts';
+
+/**
+ * A navigation card only displayed when an error occurs.
+ * @param {string} links optional links display in the card
+ * @param {string} title the title for the navigation card
+ */
+const ErrorNavCard = ({
+  iconClasses = 'vads-u-margin-right--1p5',
+  title,
+  code,
+}) => {
+  const slug = `mhv-c-card-${title.replaceAll(/\W+/g, '-').toLowerCase()}`;
+  return (
+    <div
+      className={classnames(
+        'vads-u-height--full',
+        'vads-u-padding-x--4',
+        'vads-u-padding-top--3',
+        'vads-u-padding-bottom--2',
+        'vads-u-background-color--gray-lightest',
+      )}
+    >
+      <div className="vads-l-row vads-u-margin-right--neg1">
+        <div className="vads-u-display--flex vads-u-align-items--center">
+          <div className={`vads-u-flex--auto ${iconClasses}`}>
+            <va-icon icon="error" size={4} />
+          </div>
+
+          <div className="vads-u-flex--fill vads-u-margin-right--1p5">
+            <h2 className="vads-u-margin--0" id={slug}>
+              {title}
+            </h2>
+          </div>
+        </div>
+      </div>
+
+      <p
+        className={classnames(
+          'vads-u-padding-left--0',
+          'vads-u-margin-top--2',
+          'vads-u-margin-bottom--0',
+          'vads-u-font-size--sm',
```

Comment:
The error text is using small font size. Not sure if that's the intended font size.
<img width="537" alt="Screenshot 2024-12-18 at 11 23 15 AM" src="https://github.com/user-attachments/assets/0d814d31-fea2-434b-871d-a08737cf032c" />

Reply:
It was intended. Font size based on the [figma mockup](https://www.figma.com/design/m992k2m1DSl9MXV9hDytsQ/MHV-Account-Security-%26-Sign-In?node-id=229-3&node-type=canvas&t=j4S99QhVlcTdgsgq-0).

Reply:
@kjsuarez - just double check the figma mockup and looks like the error text font size is similar font size to the `Need help?` body text which uses `vada-u-font-size--md`. 
Might be better  to use `vada-u-font-size--md` for error text as well.